src/connector.ts

```diff
@@ -144,18 +144,28 @@ export class Connector {
 	}
 
 	private setupReloadDataKoaEndpoint(): void {
-		this.koaRouter.get('/reloadData/:rundownName', async (ctx, next): Promise<void> => {
-			const RUNDOWN_EXTERNAL_ID_SUFFIX = '_1'
+		const KOA_PORT: number = 3007
+		const RUNDOWN_EXTERNAL_ID_SUFFIX: string = '_1'
+
+		this.koaRouter.post('/reloadData/:rundownName', async (context, next): Promise<void> => {
+			const rundownName: string = context.params.rundownName
 			if (!this.iNewsFTPHandler.iNewsWatcher) {
+				context.status = 503
+				context.response.body = 'Error: iNewsWatcher is undefined'
 				return
 			}
-			this.iNewsFTPHandler.iNewsWatcher?.ResyncRundown(ctx.params.rundownName + RUNDOWN_EXTERNAL_ID_SUFFIX)
 
-			await next()
+			try {
+				context.response.body = `Attempting to reload rundown with name ${rundownName}`
+				await this.iNewsFTPHandler.iNewsWatcher.ResyncRundown( rundownName + RUNDOWN_EXTERNAL_ID_SUFFIX)
```

Comment:
Are you running lint? 
A lot of your PRs (not just this one) has small syntax 'issues'. Here it is an extra space before `rundownName`.
In your PR for SofieServer for this it appears random whether your imports have spaces or not.

Reply:
I've never had this issue before with linting, i will be sure to check and do it.